From bunk@stusta.de Wed Jul 27 11:45:57 2005
Date: Wed, 27 Jul 2005 20:34:18 +0200
From: Adrian Bunk <bunk@stusta.de>
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Subject: SPEAKUP: arch/mips/Kconfig mustn't include drivers/char/speakup/Kconfig
Message-ID: <20050727183417.GJ3160@stusta.de>

This broke configuring on mips since drivers/char/speakup/Kconfig 
already gets included through drivers/Kconfig.


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/mips/Kconfig |    2 --
 1 file changed, 2 deletions(-)

--- gregkh-2.6.orig/arch/mips/Kconfig	2005-09-12 10:41:48.000000000 -0700
+++ gregkh-2.6/arch/mips/Kconfig	2005-09-12 10:41:49.000000000 -0700
@@ -931,8 +931,6 @@ config SIBYTE_STANDALONE
 	depends on SIBYTE_SB1xxx_SOC && !SIBYTE_CFE
 	default y
 
-source "drivers/char/speakup/Kconfig"
-
 config SIBYTE_STANDALONE_RAM_SIZE
 	int "Memory size (in megabytes)"
 	depends on SIBYTE_STANDALONE
